src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
+    }
+
+    /**
+     * @return string
+     */
+    public function getCurrentConversation()
+    {
+        if (is_null($this->currentConversation)) {
+            $cacheId = $this->getId() . '_current_conversation';
+            if (($currentConversation = $this->cache->fetch($cacheId)) !== false) {
+                $this->currentConversation = $currentConversation;
+
+                return $currentConversation;
+            };
+
+            $this->setCurrentConversation(HelloConversation::class);
```

Comment:
@irazasyed It's like a starting point. But it definetely does not to be hardcoded.
I see two ways of solving this:
- Set default (or starting) conversation elsewhere in application (e.g. in standalone or laravel bot)
- Let users overwrite ConversationBus::handler() method